sdk/src/test/AllTests.ls

```diff
@@ -30,9 +30,11 @@ package {
      */
     public class AllTests {   
         public static function main() {
-            var result:TestResult = TestRunner.runAll((AllTests as Type).getAssembly());
-            
-            Process.exit(result.typeReport.successful ? 0 : 1);
+            TestRunner.onComplete += function(result:TestResult) {
+                
+                Process.exit(result.typeReport.successful ? 0 : 1);
+            };
+            TestRunner.runAll((AllTests as Type).getAssembly(), true);
```

Comment:
Alright, I get what you're saying. I just worry that changing the system from Static to Object based would break backwards compatability, which this update aims to preserve.

As a compromise, it should be possible to queue test runs for different assemblies if necessary. Though, I wonder if that would really ever come into play. Unit tests are meant to all be run every time, and the unit test framework really shouldn't be used in release code